mkosi/distributions/centos.py

```diff
@@ -89,32 +94,6 @@ def install(cls, state: MkosiState) -> None:
 
         setup_dnf(state, repos)
 
-        if state.config.distribution == Distribution.centos:
-            env = dict(DNF_VAR_stream=f"{state.config.release}-stream")
-        else:
-            env = {}
-
-        invoke_dnf(state, "install", ["filesystem", *state.config.packages], env)
-
-        syslog = state.root.joinpath("etc/systemd/system/syslog.service")
-        if release <= 8 and syslog.is_symlink():
-            syslog.unlink()
-
-        # On Fedora, the default rpmdb has moved to /usr/lib/sysimage/rpm so if that's the case we need to
-        # move it back to /var/lib/rpm on CentOS.
-        move_rpm_db(state.root)
-
-        # Centos Stream 8 and below can't write to the sqlite db backend used by
-        # default in newer RPM releases so let's rebuild the DB to use the old bdb
-        # backend instead. Because newer RPM releases have dropped support for the
-        # bdb backend completely, we check if rpm is installed and use
-        # run_workspace_command() to rebuild the rpm db.
-        if release <= 8 and state.root.joinpath("usr/bin/rpm").exists():
-            cmdline = ["rpm", "--rebuilddb", "--define", "_db_backend bdb"]
```

Comment:
Considering how often mucking around with this has bitten us, this should definitely be mentioned in the NEWS.